test/integration/utils/index.js

```diff
@@ -24,4 +25,37 @@ module.exports = {
 		filename: `${__dirname}/integrationTestsLogger.logs`,
 		echo: 'log',
 	}),
+	getOpenConnections: (ports, cb) => {
+		// lsof -i :5000 -i :5001 -P -n | wc -l
+		childProcess.exec(
+			`lsof ${ports.map(p => `-i :${p}`).join(' ')} -P -n | wc -l`,
+			(err, stdout) => {
+				cb(err, parseInt(stdout.toString().trim()));
+			}
+		);
+	},
+	getIncomingConnections: (ports, cb) => {
+		// lsof -i :5000 -i :5001 -P -n -s TCP:LISTEN -t | wc -l
+		// -t to strip the headers of lsof so we can count the rows
+		childProcess.exec(
+			`lsof ${ports
+				.map(p => `-i :${p}`)
+				.join(' ')} -P -n -s TCP:LISTEN -t | wc -l`,
+			(err, stdout) => {
+				cb(err, parseInt(stdout.toString().trim()));
+			}
+		);
+	},
+	getOutgoingConnections: (ports, cb) => {
```

Comment:
Should this one be named `getEstablishedConnections`?

Reply:
Already renamed.